satpy/readers/hrit_jma.py

```diff
@@ -155,29 +266,47 @@ def get_area_def(self, dsid):
                      'proj': 'geos',
                      'units': 'm'}
 
+        area_name = '{} {}'.format(self.platform, AREA_NAMES[self.area_id])
         area = geometry.AreaDefinition(
-            "FLDK",
-            "HRIT FLDK Area: {}".format(self.projection_name),
-            'geosmsg',
-            proj_dict,
-            ncols,
-            nlines,
-            area_extent)
+            area_id=area_name.replace(' ', '-').lower(),
```

Comment:
This does not produce the same "FLDK" name that was here before. I'm not really a fan of super long area identifiers, this one for AHI would be "himawari-8-full-disk". I'm not a huge fan of the platform name being in there. Lastly, I don't think the YAML files are configured properly for all bands for your new readers so they will end up with unknown area names. We need to make sure that the H8 AHI area names work.

Reply:
Oh yes, the readers are indeed not correct, I'll fix that. And I agree that an ID should be short. What about ``FLDK``, ``NH`` and ``SH`` for full disk, northern hemisphere and southern hemisphere, respectively? I would then also remove the platform from the area name.

Reply:
That works for me. The `NH` and `SH` could be longer but I can't think of a better one that isn't kind of confusing. `south_hem`, `south_hemisphere`, `shem`, :man_shrugging: